software/apps/radio_module/cellular_test/main.c

```diff
@@ -0,0 +1,47 @@
+#include <string.h>
```

Comment:
A couple two line READMEs that explains what should happen in these tests would be great.